code/__DEFINES/colors.dm

```diff
@@ -77,6 +77,8 @@
 #define LIGHT_COLOR_FIRE         "#faa019" //Warm orange color, leaning strongly towards yellow. rgb(250, 160, 25)
 #define LIGHT_COLOR_FLARE        "#fa644b" //Bright, non-saturated red. Leaning slightly towards pink for visibility. rgb(250, 100, 75)
 #define LIGHT_COLOR_GHOST_CANDLE "#a2fad1" // Used by ghost candles. rgb(162, 250, 209)
+#define LIGHT_COLOR_PLASMA       "#2be4b8" // Used in plasma gun.
+#define LIGHT_COLOR_PLASMA_OC    "#e88893" // Used in plasma gun overcharge mode.
```

Comment:
#define LIGHT_COLOR_PLASMA       "#2be4b8" // Used in plasma gun. rgb(43, 228, 184)
#define LIGHT_COLOR_PLASMA_OC    "#e88893" // Used in plasma gun overcharge mode. rgb(232, 136, 147)